tests/org.jboss.tools.teiid.ui.bot.test/src/org/jboss/tools/teiid/ui/bot/test/ConsumeRestWs.java

```diff
@@ -174,11 +174,11 @@ public void testXmlHttpDigest() {
 	public void testHttpDigestJson() {
 		String cp = "REST_JSON_DIGEST";
 		RestConnectionProfileWizard.openWizard(cp)
-				.setConnectionUrl("http://ws-dvirt.rhcloud.com/dv-test-ws-digest/rest/json")
-				.setType(RestConnectionProfileWizard.TYPE_JSON)
-				.setAuth(RestConnectionProfileWizard.AUTH_TYPE_DIGEST, "digest", "digest")
-				.testConnection()
-				.finish();
+                .setConnectionUrl("http://dvqe05.mw.lab.eng.bos.redhat.com:8080/web-app/rest/secure/json")
+                .setType(RestConnectionProfileWizard.TYPE_JSON)
+                .setAuth(RestConnectionProfileWizard.AUTH_TYPE_DIGEST, "user", "user")
```

Comment:
It would be nice to set credentials via some properties, e.g. via reddeer config file